docs/pages/api.rst

```diff
@@ -312,6 +312,18 @@ Signature: ``Collection::asyncMapN(callable ...$callbacks): Collection;``
     $collection = Collection::fromIterable(['c' => 3, 'b' => 2, 'a' => 1])
         ->asyncMapN($mapper1, $mapper2); // ['a' => 2, 'b' => 4, 'c' => 6]
 
+averages
+~~~~~~~~
+
+Calculate the average of a list of numbers.
```

Comment:
could you add a few more details to clarify what this means exactly?

I eventually understood after staring at the code and the example for a bit, but it's not immediately clear 😅.

Maybe we could mention that it's using `scanLeft1` under the hood and produces a list in the same fashion.

Reply:
Done!